src/ToolInfo.php

```diff
@@ -73,31 +72,31 @@ public static function isInstalledByComposer()
         static $result;
 
         if (null === $result) {
-            $result = !self::isInstalledAsPhar() && file_exists(self::getScriptDir().self::COMPOSER_INSTALLED_FILE);
+            $result = !self::isInstalledAsPhar() && file_exists(self::getComposerInstalledFile());
         }
 
         return $result;
     }
 
-    private static function getScriptDir()
+    private static function getComposerInstalledFile()
     {
         static $result;
 
         if (null === $result) {
-            $script = $_SERVER['SCRIPT_NAME'];
-
-            if (is_link($script)) {
-                $linkTarget = readlink($script);
-
-                // If the link target is relative to the link
-                if (false === realpath($linkTarget)) {
-                    $linkTarget = dirname($script).'/'.$linkTarget;
+            $composerInstalledFile = '/composer/installed.json';
+            $checkedPaths = array(
+                // this project itself at ./src
+                __DIR__.'/../vendor'.$composerInstalledFile,
```

Comment:
Not sure we should check this one actually: the project does not require itself via Composer so I think the version won't be available in the json file.